common/i18n/resources/en.js

```diff
@@ -308,6 +317,17 @@ module.exports = {
     }
   },
 
+  dateUtils: {
+    aMomentAgo: 'A moment ago',
+    hour: 'hour',
+    hour_plural: 'hours',
+    day: 'day',
+    day_plural: 'days',
+    week: 'week',
+    week_plural: 'weeks',
+    timeDiff: `{{diff}} $t(dateUtils.{{unit}}, { 'count': 3 }) ago`,
```

Comment:
why count is 3 by default? what about 1 week ago?

Reply:
Right